ReadMe.md

```diff
@@ -17,6 +17,12 @@ For quick questions jump on Gitter and ask away.
 
 For bugs, make sure there isn't an active issue and then create one.
 
+## Why this library and not FastLED or otherlibraryY?
+See [Why this Library in the Wiki](https://github.com/Makuna/NeoPixelBus/wiki#why-this-library). 
+The main plus of this library is using the UART to do drive the pixels via DMA (i.e. without interrupts) 
```

Comment:
Please remove everything after the link.  I want nothing here that has any details that may change; like FastLED may add RGBW in the future or I may add RMT support. 

I would make the section header like

## Library Comparisons
[Choosing a NeoPixel Library](wiki link)

Reply:
Also note, I did a little change to your wiki editing, again, I try to minimize top page sizes and create specific ones.  So your link will need to change